### PR TITLE
fixing knimin/tests/test_ag_new_barcode.py

### PR DESCRIPTION
**Pull request by josenavas**
@sjanssen2 @antgonza able for a quick review?

### CODE REVIEW

File: knimin/tests/test_ag_new_barcode.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def test_post(self):
         self.mock_login_admin()
         action = 'unknown'
         num_barcodes = 4
-        projects = ["American Gut Project", "Autism Spectrum Disorder"]
+        projects = ["American Gut Project"]
```
Comment (sjanssen2) [inline knimin/tests/test_ag_new_barcode.py]: I'd recommend to test against a utf8 encoded project name.

File: knimin/tests/test_ag_new_barcode.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ def test_post(self):
         self.assertEqual(response.headers['Content-Disposition'],
                          'attachment; filename=barcodes.pdf')
         # check that the files is a PDF ...
-        self.assertIn('%PDF-1.4', response.body)
+        self.assertIn('%PDF-1.', response.body)
```
Comment (antgonza) [inline knimin/tests/test_ag_new_barcode.py]: Why this change? Making sure that any 1* version works?

Comment (josenavas) [inline knimin/tests/test_ag_new_barcode.py]: Yes, in my machine it was failing because my PDF is 1.5

**Comment by josenavas** [comment]
There was some stochasticity in one of the tests that I just fixed - @antgonza do you want to make the honors?
